.travis.yml

```diff
@@ -139,6 +140,25 @@ matrix:
           branch_pattern: coverity_scan
       after_failure:
         - cat /home/travis/build/CVC4/CVC4/cov-int/build-log.txt
+    - os: osx
+      jdk: openjdk-7-jdk
+      before_install:
+      - |
+        contrib/mac-build
+        brew install swig
```

Comment:
Swig _could_ be installed via `contrib/mac-build`, but I wasn't sure if there was a reason for it to be omitted, so I added it here instead.